project/plugins.sbt

```diff
@@ -1,7 +1,16 @@
 logLevel := Level.Warn
 
+// Typesafe Resolvers
 resolvers += Resolver.typesafeRepo("releases")
 
+// S3 based SBT resolver
+resolvers ++= Seq(
+  "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com",
+  "HAT Library Artifacts Snapshots" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-snapshots.hubofallthings.com",
+  "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com"
```

Comment:
Releases resolver is duplicated between lines 8 and 10. This line can be safely removed.